spec/requests/api/v1/exercises_spec.rb

```diff
@@ -126,5 +126,31 @@
         end
       end
     end
+
+    describe '#update' do
+      let(:url) { '/api/v1/exercises/' }
+
+      context 'with valid params' do
+        before do
+          @exercise = create(:exercise)
+          @initial_exercise_count = Exercise.count
+          params = { exercise: { name: Faker::Name.name, description: Faker::Lorem.paragraph } }
+          put "#{url}#{@exercise.id}", params: params
+        end
+
+        it 'updates the record' do
+          expect(response).to have_http_status('200')
+          expect(Exercise.count).to eq(@initial_exercise_count)
+        end
```

Comment:
It could be useful to add a small check to see that the fields have actually updated

Reply:
Have updated with some additional expects